invenio_accounts/alembic/62efc52773d4_create_useridentity_table.py

```diff
@@ -0,0 +1,54 @@
+#
```

Comment:
Note: Another migration is to be added to Invenio-OAuthclient that moves the data from the previous table. Imports is kept backward compatible in invenio-oauthclient